catalogs/catalog-jdbc-doris/src/main/java/com/datastrato/gravitino/catalog/doris/operation/DorisTableOperations.java

```diff
@@ -194,6 +192,55 @@ static void appendIndexesSql(
     sqlBuilder.append(",").append(NEW_LINE).append(indexSql);
   }
 
+  private static void appendPartitionSql(
+      Transform[] partitioning, JdbcColumn[] columns, StringBuilder sqlBuilder) {
+    if (ArrayUtils.isEmpty(partitioning)) {
+      return;
+    }
+
```

Comment:
Since we only use `partitioning[0]` below, the preconditions check of `partitioning` length should be added

Reply:
Precondition added